assets/src/pages/webrtcInternals/hooks/useDetailsToggle.tsx

```diff
@@ -0,0 +1,33 @@
+import { useCallback, useState } from "react";
+
+type DetailsToggleApi = {
+  openKeys: Set<string>;
+  isOpen: (key: string) => boolean;
+  toggle: (key: string) => void;
+};
+
+const useDetailsToggle = (): DetailsToggleApi => {
```

Comment:
I replaced this hook with `useToggle` (description `InternalsSection.tsx`). If my reasoning is right please remove this file.